controllers/nginx/rootfs/Dockerfile

```diff
@@ -12,21 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM BASEIMAGE
-
-CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
+FROM gcr.io/google_containers/nginx-slim-amd64:0.23
```

Comment:
Ya. I was having issues it picking up the `BASEIMAGE` defined in the makefile so I explicitly stated it here